Tests/SwiftGenTests/ConfigLintTests.swift

```diff
@@ -187,9 +187,9 @@ class ConfigLintTests: XCTestCase {
         (.error, "xcassets.inputs: Fixtures/XCAssets/Images.xcassets does not exist"),
         (.error, "xcassets.inputs: Fixtures/XCAssets/Images.xcassets does not exist"),
         (.error, """
-          xcassets.outputs: Template named custom-swift3 not found. Use `swiftgen templates` to \
-          list available named templates or use `templatePath` to specify a template by its full \
-          path.
+          xcassets.outputs: Template named custom-swift3 not found. Use `swiftgen templates list` \
```

Comment:
Wait, to we still have to use `\` at the end of line even if we're using Swift 4's multi-line string literals?

Reply:
It’s to keep it as one line

Reply:
Ah yeah of course 👍